### PR TITLE
Create memes.md

### PR DESCRIPTION
**Pull request by ooyce**
doggo memes

### CODE REVIEW

File: memes.md [hunk 1]
```diff
@@ -0,0 +1 @@
+assistant to the branch manager
```
Comment (strawhat-123) [inline memes.md]: it should be:
```
branch manager
assistant to the branch manager
```

**Review by strawhat-123 — CHANGES_REQUESTED** [review]
branch

File: memes.md (hunk 1 above)
Comment (ooyce) [reply memes.md]: okie dokie

**Review by strawhat-123 — APPROVED** [review]
looks goot now

**Review by strawhat-123 — APPROVED** [review]
brilliant

**Pull request merged**